### PR TITLE
Feature/update postgres

### PR DESCRIPTION
**Pull request by 4c74356b41**
postgres moved to a latest bitnami chart.
namespace resource deleted for 2 reasons:
1. postgres would be deployed to default namespace with the current setup
2. this should be handled by helm using `--create-namespace` 

also: started work to unify all the certificates into common secrets to avoid certificate sprawl caused by certs folders under every chart for no good reason; added example how to run this new code under dev-local-rework values file.

a command to test this locally would be something like this:
```
helm upgrade --install tip-wlan . -f .\resources\environments\kind.yaml.local_dev -n tip --create-namespace
```

### CODE REVIEW

**Review by eugenetaranov-opsfleet — APPROVED** [review]

File: tip-wlan/Chart.yaml [hunk 1]
```diff
@@ -26,54 +26,55 @@ sources:
...
+- name: kafka
+  version: 0.1.0
+  condition: kafka.enabled
+- name: postgresql
+  version: 10.1.0
+  repository: https://charts.bitnami.com/bitnami
+  condition: postgresql.enabled
+- name: cassandra
+  version: 5.5.3
+  condition: cassandra.enabled
```
Comment (rahulsharma2020) [inline tip-wlan/Chart.yaml:80]: Are we missing repository here?

Comment (4c74356b41) [reply tip-wlan/Chart.yaml:80]: this is a postgres PR, cassandra PR has this.

**Comment by leonidlm** [comment]
@4c74356b41 for this to run, do we need to place the SSL certificates for PostgreSQL into another folder? If that's the case, we should probably update the https://github.com/Telecominfraproject/wlan-pki-cert-scripts/blob/master/copy-certs-to-helm.sh script, otherwise merging this will break the environment for everyone, no?

**Comment by 4c74356b41** [comment]
yeah, thats true @leonidlm, but we'd just pile all the certificates into a single place, thats the idea behind shared secrets, so you dont even need the script anymore. I mean, for now you still need the script, because many old parts still use it, but for the new code to work just copy all the certs to the new certs location

**Comment by 4c74356b41** [comment]
https://github.com/Telecominfraproject/wlan-cloud-helm/pull/49

**Pull request closed without merging**